--- include/prereq-build.mk	2016-06-12 11:30:27.000000000 +0000
+++ /opt/prereq-build.mk	2016-06-12 23:11:37.098500000 +0000
@@ -59,7 +59,7 @@
 
 define Require/ncurses
 	echo 'int main(int argc, char **argv) { initscr(); return 0; }' | \
-		gcc -include ncurses.h -x c -o $(TMP_DIR)/a.out -lncurses -
+		gcc -include ncurses.h -x c -o $(TMP_DIR)/a.out - -lncurses
 endef
 
 $(eval $(call Require,ncurses, \
@@ -69,7 +69,7 @@
 
 define Require/zlib
 	echo 'int main(int argc, char **argv) { gzdopen(0, "rb"); return 0; }' | \
-		gcc -include zlib.h -x c -o $(TMP_DIR)/a.out -lz -
+		gcc -include zlib.h -x c -o $(TMP_DIR)/a.out - -lz
 endef
 
 $(eval $(call Require,zlib, \
